SECURITY.md

```diff
@@ -0,0 +1,46 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Security Policy](#security-policy)
+  - [Security](#security)
+  - [Reporting Security Issues](#reporting-security-issues)
+  - [Additional Information](#additional-information)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
```

Comment:
Why do we have this TOC header on this repo and not the other ones?

Reply:
It was added automatically by a pre-commit hook. And it seems there's no way to skip this (I've tried the skip as the documentation suggested without luck).

Reply:
Oh, I see, this repo has an extra pre-commit rule named doctoc.